aws/data_source_aws_eks_cluster_auth_test.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSEksClusterAuthDataSource_basic(t *testing.T) {
+	dataSourceResourceName := "data.aws_eks_cluster_auth.test"
+
+	resource.Test(t, resource.TestCase{
```

Comment:
Nit: We should use `resource.ParallelTest()` instead of `resource.Test()` here.

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```